### PR TITLE
Add ndm.app 0.0.3-beta

### PR DESCRIPTION
**Pull request by 45kb**
*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked that the cask was not already refused in [closed issues].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed

Add ndm.app 0.0.3-beta

### CODE REVIEW

**Review by vitorgalvao — APPROVED** [review]

File: Casks/ndm.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  version '0.0.3-beta'
+  sha256 '19055224bbbc3dd36f272ef552142b49425d8a2fffda2b407045b8d8f6257ec6'
+
+  # github.com/720kb/ndm was verified as official when first introduced to the cask
+  url "https://github.com/720kb/ndm/releases/download/#{version}/ndm-#{version}.dmg"
+  appcast 'https://github.com/720kb/ndm/releases.atom',
+          checkpoint: '0db6ec7462676831c0922f595d653df6f903a49a540d41c81f0b6f5e1c2212b1'
+  name 'ndm'
+  homepage 'https://720kb.github.io/ndm'
+  license :gpl
```
Comment (miccal) [inline Casks/ndm.rb]: Please remove the `licence` stanza, it has been deprecated.

**Review by miccal — CHANGES_REQUESTED** [review]

File: Casks/ndm.rb (hunk 1 above)
Comment (45kb) [inline Casks/ndm.rb]: @miccal sorry, ok i'll rebase and fix it.

Comment (45kb) [inline Casks/ndm.rb]: @miccal ouch i see many commits, is everythings ok?

Comment (vitorgalvao) [inline Casks/ndm.rb]: > i see many commits, is everythings ok?

Problem occurred with your rebasing. Please don’t squash cask changes.

Comment (vitorgalvao) [inline Casks/ndm.rb]: > i see many commits, is everythings ok?

Problem occurred with your rebasing. Please don’t squash cask changes.

You may submit a new PR if that’d be easier to fix.

**Pull request closed without merging**